README.md

```diff
@@ -25,6 +25,34 @@ The following must be installed in order run Jekyll locally.
 6. Open <http://localhost:4000/> to view the site
 7. When finished hit `ctrl + c` in the terminal window to stop the server
 
+### API Documentation Development Setup
+
+The API documentation is generated from [sparkpost-api-documentation-DEV](https://github.com/SparkPost/sparkpost-api-documentation-DEV].  See the [README](https://github.com/SparkPost/sparkpost-api-documentation-DEV/blob/master/README.md) in that repo for details on that build process.
+
+For an easy dev setup:
+
+- Check out sparkpost-api-documentation-DEV
+- Check out sparkpost.github.io alongside
+- Start the API docs watch-and-build process:
+
+```bash
+cd sparkpost-api-documentation-DEV
+grunt staticDev
```

Comment:
This flow works, but might not be super simple for devs that are used to the existing setup. As discussed on Slack, maybe we can have this command check out the dev hub? Though the dev hub does require ruby and bundler to work. Open to other options.